FIPS/src/test/java/net/snowflake/client/RunningOnGCP.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
```

Comment:
let's change the years, also in other files

Reply:
@sfc-gh-dprzybysz Done